src/containers/PipelineRuns/PipelineRuns.js

```diff
@@ -75,6 +94,90 @@ export /* istanbul ignore next */ class PipelineRuns extends Component {
     this.setState({ showCreatePipelineRunModal });
   };
 
+  handleChange = event => {
+    const inputValue = event.target.value;
+    this.setState({
+      currentFilterValue: inputValue
+    });
+  };
+
+  handleAddFilter = event => {
+    event.preventDefault();
+    const currentURL = this.props.match.url;
+    const { currentFilterValue = '' } = this.state;
+    const filterValue = currentFilterValue.replace(/\s/g, '');
+    if (!filterValue.match('([a-z0-9A-Z-_./]:[a-z0-9A-Z-_.],?)+')) {
+      this.setState({
+        validFilter: {
+          state: false,
+          filterMessageId: 'dashboard.pipelineRuns.invalidFilter',
+          filterDefaultMessage:
+            'Filters must be of the format labelKey:labelValue and use only accepted label characters'
```

Comment:
contain instead of use

Again would mention the regex, and what is state being false here again? It's invalidFilter right

Reply:
we decided on using positive values for readability so isValid = true / isValid = false hence an invalidFilter